types/validator.yaml

```diff
@@ -76,6 +76,23 @@ ProposerDuty:
         - $ref: "./primitive.yaml#/Uint64"
         - description: "The slot at which the validator must propose block."
 
+Altair:
+  SyncDuty:
+    type: object
+    properties:
+      pubkey:
+        $ref: './primitive.yaml#/Pubkey'
+      validator_index:
+        allOf:
+          - $ref: "./primitive.yaml#/Uint64"
+          - description: "Index of validator in validator registry."
+      sync_committee_indices:
```

Comment:
Can we consider renaming this field?

If it stays then the name `sync_committee_indices` will be overloaded to mean:

1) The validator indices of a sync committee
2) The positions of a single validator index in a sync committee

Given that these two meanings appear together frequently it's quite confusing. I think a better name for the 2nd definition would be `sync_committee_positions`, although I'm also open to other suggestions. Another option would be `validator_sync_committee_indices`, in keeping with the similarly named `validator_committee_index` for attestation duties.